homeworks/karolinavalko/5_oop/modules.py

```diff
@@ -0,0 +1,72 @@
+class Car:
+
+    def __init__(self, car_brand: str, car_model: str, production_year: int):
+        self.car_brand = car_brand
+        self.car_model = car_model
+        self.production_year = production_year
+        self.car_mileage = 0
+        self.car_fuel_level = 100
+
+    def __str__(self):
+        return f"{self.car_brand}, {self.car_model}, {self.production_year}"
+
+    def drive(self, distance: float):
```

Comment:
itt is szuper lenne megnézni hogy negatív e a distance és egyből hibát dobni.

Reply:
Javitottam :-)